cli/command/secret/cmd.go

```diff
@@ -11,7 +11,7 @@ import (
 func NewSecretCommand(dockerCli command.Cli) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "secret",
-		Short: "Manage Docker secrets",
+		Short: "Manage Swarm secrets",
```

Comment:
Complementary of https://github.com/docker/cli/pull/3309

Reply:
Ah, yes. I wasn't entirely sure to put `Swarm` on everything (including, e.g. `List Swarm secrets` etc), but I'm planning to take a closer look at the "long" descriptions, and make some of those slightly more useful.